client/schemas/company.ts

```diff
@@ -69,6 +76,14 @@ export interface Company {
     final: PartnerVoteObj[];
     prevote: PartnerVoteObj[];
   };
+  voteResults?: {
```

Comment:
Are we putting this in the database? If not can you add a JSDoc comment saying it’s a server field?